core/src/test/resources/json-ld-api-tests/README.md

```diff
@@ -0,0 +1,106 @@
+# Introduction
```

Comment:
I see, this is a copy of https://github.com/w3c/json-ld-api/blob/master/tests/README.md.
Will add a remark in a commit - I think it is helpful to know the source?

Reply:
It is indeed helpful to know the source. However, the workflow for updating the tests (as documented in the README, 'Update local spec tests') involves copying `json-ld-api/tests/*` to `json-ld-api-tests/`, which replaces the README. Maybe we could add a separate README to `src/test/resources`, describing the origins of the spec tests?